src/Analytics/Renderer/Google/GoogleClassicAnalytics.php

```diff
@@ -52,7 +52,12 @@ public function getScript()
         $script[] = '<script>';
 
         foreach ($this->participations as $testIdentifier => $variationIndex) {
-            $script[] = "cxApi.setChosenVariation(" . (int) $variationIndex . ", '" . (string) $testIdentifier . "')";
+            $script[] = "cxApi.setChosenVariation(" . (int) $variationIndex . ", '" . (string) $testIdentifier . "');";
+        }
+
+        if (true === $this->getEventTriggerInclusion()) {
+            $script[] = "_gaq.push(['_trackEvent', 'PhpAb', 'testRun', 'testsAmout', "
```

Comment:
Maybe we should make this configurable aswell. Users might not want this name or maybe they want to hide that they are using PHP (or our library in particular).